spec/integration/game_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rspec'
+require 'rspec/core'
```

Comment:
I don't think you should need to do this in each test file. Perhaps if you run your specs with `bundle exec` you won't need the requires.